src/components/ChallengeModal/ChallengeModal.tsx

```diff
@@ -1749,7 +1758,7 @@ export class ChallengeModalBody extends React.Component<
                 width: this.state.challenge.game.width,
                 height: this.state.challenge.game.height,
                 ranked: true,
-                handicap: this.state.challenge.game.handicap !== "0",
+                handicap: this.state.challenge.game.handicap.toString() !== "0",
```

Comment:
Typescript complained that a comparison between number and string is superfluous, so I added this conversion. I'm not sure what's going on though, did I get the type wrong, or is this comparison needed for legacy reasons?

Reply:
If you're sure this is a number, then the original code looks like a bug to me.  I'd probably just do numeric comparison:

```suggestion
                handicap: this.state.challenge.game.handicap !== 0,
```

Reply:
I see now that string values can get into state because the input is a <select>. I intend to fix it by converting the string to number before it gets into the state.